outdoorsmanship.lic

```diff
@@ -17,19 +18,21 @@ class Outdoorsmanship
     ]
     args = parse_args(arg_definitions)
 
-    settings = get_settings
+    @settings = get_settings
 
+    @training_spells = @settings.crafting_training_spells
     @targetxp = 3
     @targetxp = args.mindstates.to_i if args.mindstates
 
     @forage_item = if args.collect_item
                      args.collect_item
                    else
-                     settings.forage_item
+                     @settings.forage_item
                    end
 
     start_exp = DRSkill.getxp('Outdoorsmanship')
     @end_exp = [start_exp + @targetxp, 29].min
+    Flags.add('outdoors-magic-ready', 'You feel fully prepared to cast your spell.')
```

Comment:
💡 We should probably establish some standard like "every script has a setup function which checks args, adds flags, get settings, ..." and "every script has a before_dying if it needs to delete flags, ..."

Reply:
very much agreed going forward, i submitted this before doing the before_dying thing.

Reply:
it makes sense to always delete the flags if the script dies even if it dies from being killed instead of just a graceful exit.

Reply:
added this to the review document in https://github.com/rpherbig/dr-scripts/wiki/Things-to-look-for-in-code-review